README.md

```diff
@@ -1,2 +1,78 @@
+# Overview
+Charmed Ceph is a software-defined storage solution that provides object, block and file storage on commodity hardware.
+
+Charmed Ceph makes it easy to use and deploy [Ceph](https://ceph.com/en/). It provides a simpler way to deliver Ceph to users using a set of scripts called charms that are deployed with [Juju](https://juju.is/).
+
+Corporate users can benefit from the robust and manageable full-suite storage solution, and the ease with which their Ceph clusters can be deployed with Charmed Ceph.
+
+# Repository Contents
+
 # ceph-charms
-Monorepo for ceph charms
+
+This is a monolithic repository, or monorepo, that contains the core charms used in Ceph deployment.
+It also contains other associated charm libraries and utilities that make up the Ceph ecosystem that
+can be deployed via Juju.
+
+These charms used to reside in their own repository in the past. The reason they have been consolidated
+into a single repository is to make larger changes that involve multiple components easier.
+
+There are nine individual charms in this monorepo, each defining and deploying a specific Ceph application.
+However, these charms often work together, i.e. they are often deployed together via Juju integrations.
+For example, the ceph-osd and ceph-mon charms are often deployed together to provide and scale storage in a Ceph cluster.
+
+# The charm index
+
+1. [ceph dashboard](https://github.com/canonical/ceph-charms/tree/main/ceph-dashboard)
+2. [ceph-fs](https://github.com/canonical/ceph-charms/tree/main/ceph-fs)
+3. [ceph-mon](https://github.com/canonical/ceph-charms/tree/main/ceph-mon)
+3. [ceph-nfs](https://github.com/canonical/ceph-charms/tree/main/ceph-nfs)
+5. [ceph-nvme](https://github.com/canonical/ceph-charms/tree/main/ceph-nvme)
+6. [ceph-osd](https://github.com/canonical/ceph-charms/tree/main/ceph-osd)
+7. [ceph-proxy](https://github.com/canonical/ceph-charms/tree/main/ceph-proxy)
+8. [ceph-radosgw](https://github.com/canonical/ceph-charms/tree/main/ceph-radosgw)
+9. [ceph-rbd-mirror](https://github.com/canonical/ceph-charms/tree/main/ceph-rbd-mirror)
+
+# Additional sub-projects
+
+In addition to charms, the repository contains the following:
```

Comment:
[* nit/ suggested rewording*] In addition to _the_ charms, the monorepo also contains libraries, dependencies, modules and test scripts, including: